developer/reference/runtime/register-component.mdx

```diff
@@ -146,6 +149,67 @@ runtime.registerComponent(Cube, {
 });
 ```
 
+### Adding a description
+
+This example shows how to add an `Circle` component with a rich description. We
+use markdown formatting for our tooltip descriptions.
+
+In the builder component tray and panel, we can see an info icon next to the component label.
+
+<Frame>
+  <img
+    src="/images/register-custom-desc1.png"
```

Comment:
Can we make this bigger and also put this in a more specific folder. I would suggest `/images/developer/reference/register-component/`.

Reply:
Actually looking at this again. I think we can remove this image completely since the info icon shows up in the next image as well